packages/fluentui/react-northstar/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -91,7 +91,13 @@ export const Breadcrumb = compose<'nav', BreadcrumbProps, BreadcrumbStylesProps,
         })}
       >
         <BreadcrumbContext.Provider value={contextValue}>
-          <div role="list">{childrenExist(children) ? children : content}</div>
+          <div
+            {...getA11yProps('itemsContainer', {
```

Comment:
nit: let's use just `container`

Reply:
LOL lightening review speed. Electricity still off huh? :D 
Changed as requested in the new commit.